src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+    >
+
+      <img
+        src="images/Search.svg"
+        alt="big serch"
+        class="button"
+      >
+
+      <input 
+        type="text"
+        data-qa="keypress"
+        name="big"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="serch-form"
+      >
+
+    </form>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="small"
+    >
+
+      <img
+        src="images/Search.svg"
+        alt="small serch"
+        class="button button-small"
+      >
+
+      <input type="text"
+        name="small"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="serch-form serch-form-small "
+      >
```

Comment:
please be attentive to attributes and write them from the new line each
```suggestion
      <input
        type="text"
        name="small"
        placeholder="Try &quot;Los Angeles&quot;"
        class="serch-form serch-form-small "
      >
```